docs/docs/02-quick-start/01-installation.md

```diff
@@ -10,19 +10,52 @@ go install github.com/a-h/templ/cmd/templ@latest
 
 This installs templ into your path.
 
-## go install (as tool)
+## go install (as tool) - Recommended for projects
```

Comment:
I'm not sure we do recommend this yet. It only works for Go 1.24, and we still support 2 previous versions of Go. WDYT @joerdav?

If it was the recommended way, I'd want it at the top of the page so it's the first thing you see.

Reply:
I'd be in favour of it being the recommended approach, it eliminates the problem of mismatched versions which is a common frustration I think.